Packs/iDefense/Integrations/iDefense_v2/iDefense_v2.yml

```diff
@@ -0,0 +1,169 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: iDefense_v2
+  version: -1
+configuration:
+- display: URL
+  name: url
+  defaultvalue: https://api.intelgraph.idefense.com/
+  type: 0
+  required: true
+- display: API Token
+  name: api_token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: use_proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+description: iDefense provides intelligence regarding security threats and vulnerabilities.
+display: iDefense v2
+name: iDefense_v2
+script:
+  commands:
+  - name: ip
+    arguments:
+    - name: ip
```

Comment:
as we support csv, please add is array to all relevant args, e.g:
```
    - default: false
      description: Indicators to search.
      isArray: true
      name: indicators
      required: true
      secret: false
```

Reply:
csv file? How we support it?

Reply:
not csv file, I miswrote. Is array, which is a comma separated ip values for examples.
you added support for it in the code, but need to allow it from the yml side.